src/check_pfda/core.py

```diff
@@ -1,16 +1,115 @@
 """Collect tests and run them on supplied code."""
 
+import os
+from pathlib import Path
+import sys
+import logging
+import platform
+
+
 from click import echo
+import pytest
+from check_pfda.utils import get_current_assignment, get_tests, _add_src_to_sys_path, _remove_src_from_sys_path
+
+
+logger = logging.getLogger(__name__)
+
+
+def check_student_code(verbosity: int = 2, debug=False) -> None:
+    """Main check-pfda runner. Outputs results of tests to scripts in `src` to stdout."""
+    # Configure logging
+    root_path = Path.cwd()
+    if root_path.name == "src":
+        root_path = root_path.parent
+
+    src_path = root_path / "src"
```

Comment:
```suggestion
    src_path = root_path / "src"
```

This can be a GLOBAL var too.